requirements-dev.txt

```diff
@@ -5,7 +5,6 @@ pytest-cov
 coverage==6.0.2  # see https://github.com/pytest-dev/pytest-cov/issues/520
 tqdm
 scikit-learn
-torchvision
```

Comment:
Seems like an excess dependency.

If present, it forces to upgrade torch to 2.0.0 during `pip install -r requirements-dev.txt`, which breaks installation from source (`build_and_test_p2pd` in CI).